docs/source/starter/new-project.rst

```diff
@@ -12,57 +12,110 @@
 
 .. _new_project:
 
-####################
-Lightning in 2 steps
-####################
+############
+Introduction
+############
 
-**In this guide we'll show you how to organize your PyTorch code into Lightning in 2 steps.**
+*************************
+What is PyTorch Lightning
+*************************
 
-Organizing your code with PyTorch Lightning makes your code:
+PyTorch has all you need to train your models. But there's a lot more to deep learning more than just attaching some layers together. It provides you the APIs required to build models, datasets etc.
+but when it comes to actual training, there's a lot of boilerplate code involved that you need to write by yourself and if you need to scale your training/inferencing on multiple devices/machines there's another
+set of integrations you might need to do by yourself.
+
+So here PyTorch Lightning comes into the picture. All you need is some restructuring of your existing code and set certain flags and voila, you are done.
+Now you can train your models on different accelerators like GPU/TPU/IPU, do distributed training accross multiple machines/nodes with no code change use state-of-the-art distributed training mechanisms
+and a lot more. Now all you need is to take care of your research code and leave the engineering to us.
+
+Code organization is the core of Lightning. It leaves core training and validation logic to you and automates the rest.
+
+----------
+
+********************
+Lightning Philosophy
+********************
+
+Organizing your code with Lightning makes your code:
 
 * Keep all the flexibility (this is all pure PyTorch), but removes a ton of boilerplate
 * More readable by decoupling the research code from the engineering
 * Easier to reproduce
 * Less error-prone by automating most of the training loop and tricky engineering
 * Scalable to any hardware without changing your model
 
-----------
+Lightning is built for:
 
-Here's a 3 minute conversion guide for PyTorch projects:
+* Researcher who want to focus on research without worrying about the engineering aspects of it
+* ML Engineers who want to built reproducible pipelines
+* Data Scientists who want to try out different models for their tasks and build-in ML techniques
+* Educators who seek to study and teach Deep Learning with PyTorch
 
-.. raw:: html
+The team makes sure that all the latest techniques are already integrated and maintained periodically to make sure you don't
+have to worry about any engineering stuff and just focus on research.
+
+It is tested across multiple Python/PyTorch versions to ensure its reach.
```

Comment:
I don't think this sentence fits here, there's the installation and readme for it.

```suggestion
```